README.md

```diff
@@ -45,18 +45,35 @@ the serializer generator:
 $ rails g serializer post
 ```
 
-### Support for PORO's and other ORM's.
```

Comment:
it's what i'm used to - apostrophe's signal possession or a contraction, not a plural. i can swap back if it would be confusing to users!